### PR TITLE
Add more debug logs to show what is happening

### PR DESCRIPTION
**Pull request by ruudk**
Makes it easier to debug when things go wrong.

### CODE REVIEW

File: aws/ec2.go [hunk 1]
```diff
@@ -168,6 +168,7 @@ func getSubnets(svc ec2iface.EC2API, vpcID, clusterID string) ([]*subnetDetails,
 		return nil, err
 	}
 
+	log.Debug("getRouteTables")
```
Comment (szuecs) [inline aws/ec2.go]: Should this not have `aws.` prefix?

File: aws/adapter.go [hunk 2]
```diff
@@ -588,22 +589,27 @@ func (a *Adapter) DeleteStack(stack *Stack) error {
...
 
 	clusterID := instanceDetails.clusterID()
 
+	log.Debug("findSecurityGroupWithClusterID")
 	securityGroupDetails, err := findSecurityGroupWithClusterID(awsAdapter.ec2, clusterID, awsAdapter.controllerID)
 	if err != nil {
 		return nil, err
 	}
 
+	log.Debug("getSubnets")
```
Comment (szuecs) [inline aws/adapter.go]: Similar to all new logs in this file `aws.` prefix might make sense, not?

**Comment by ruudk** [comment]
@szuecs Applied your feedback.

**Comment by mikkeloscar** [comment]
@ruudk needs rebase now :)

**Comment by ruudk** [comment]
@mikkeloscar Done

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
@ruudk needs rebase again.

**Comment by ruudk** [comment]
@mikkeloscar Done, and green now :)

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**